### PR TITLE
Np 9065 project patch definitions

### PR DESCRIPTION
**Pull request by svenngjeto**
Open API specification with samples and AWS Template for PATCH project  service

### CODE REVIEW

File: docs/cristin-proxy-swagger.yaml [hunk 1]
```diff
@@ -1937,6 +2052,22 @@ components:
           en: "Summary in an academic manner"
         popularScientificSummary:
           en: "popularScientificSummary popularScientificSummary popularScientificSummary"
+    ProjectPatchExample:
+      value:
+        title: 'Updated Title'
+        alternativeTitles:
+          - 'en': 'Updated Title in english'
+        startDate: '2022-10-09T00:00:00.001Z'
+        endDate: '2030-12-31T00:00:00.001Z'
```
Comment (torarnet) [inline docs/cristin-proxy-swagger.yaml:2061]: Does it now work with 000Z instead of 001Z?

File: docs/cristin-proxy-swagger.yaml [hunk 2]
```diff
@@ -1328,6 +1380,69 @@ components:
...
+          type: string
+          pattern: '^http:\/\/lexvo.org\/id\/iso639-3\/[a-z]{3}$'
+          description: The lexvo URI for the language of the Project
+        alternativeTitles:
+          type: array
+          description: Titles in alternative languages
+          items:
+            $ref: '#/components/schemas/Languages'
+        startDate:
+          type: string
```
Comment (torarnet) [inline docs/cristin-proxy-swagger.yaml:1400]: can this be a date-time format?

File: docs/cristin-proxy-swagger.yaml (hunk 1 above)
Comment (svenngjeto) [reply docs/cristin-proxy-swagger.yaml:2061]: Yes,  CustomInstantSerializer is supposed to take care of that -( when the model comes in place....)

File: docs/cristin-proxy-swagger.yaml [hunk 3]
```diff
@@ -110,6 +110,58 @@ paths:
...
+              objectExample:
+                $ref: '#/components/examples/ProjectPatchExample'
+      responses:
+        204:
+          description: Successful update of resource
+          content:
+            application/json:
+              schema:
+                type: string
+                example: '{}'
```
Comment (torarnet) [inline docs/cristin-proxy-swagger.yaml:149]: I think this is now not a empty json but just Void/null

**Review by torarnet — DISMISSED** [review]

**Review by torarnet — APPROVED** [review]

**Pull request merged**